src/main/java/uk/gov/hmcts/reform/sscs/mappers/ServiceHearingValuesMapper.java

```diff
@@ -0,0 +1,95 @@
+package uk.gov.hmcts.reform.sscs.mappers;
+
+import org.springframework.stereotype.Component;
+import uk.gov.hmcts.reform.sscs.ccd.domain.CcdValue;
+import uk.gov.hmcts.reform.sscs.ccd.domain.OtherParty;
+import uk.gov.hmcts.reform.sscs.ccd.domain.SscsCaseData;
+import uk.gov.hmcts.reform.sscs.ccd.domain.SscsCaseDetails;
+import uk.gov.hmcts.reform.sscs.ccd.domain.YesNo;
+import uk.gov.hmcts.reform.sscs.helper.mapping.HearingsDetailsMapping;
+import uk.gov.hmcts.reform.sscs.model.service.hearingvalues.ServiceHearingValues;
+import uk.gov.hmcts.reform.sscs.utils.SscsCaseDataUtils;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+import java.util.concurrent.atomic.AtomicReference;
+
+
+@Component
+public class ServiceHearingValuesMapper {
+
+
+    public ServiceHearingValues mapServiceHearingValues(SscsCaseDetails caseDetails) {
+        if (caseDetails == null) {
+            return null;
+        }
+
+        SscsCaseData caseData = caseDetails.getData();
+
+        return ServiceHearingValues.builder()
+                .caseName(SscsCaseDataUtils.getCaseName(caseData))
+                .autoListFlag(false) // TODO to be provided in a future story, right now not populated
+                .hearingType(SscsCaseDataUtils.getHearingType(caseData))
+                .caseType(caseData.getBenefitCode())
+                .caseSubTypes(SscsCaseDataUtils.getIssueCode(caseData))
+                // TODO same method is in HearingsDetailsMapping -> buildHearingWindow
+                //  (SSCS-10321-Create-Hearing-POST-Mapping)
+                .hearingWindow(SscsCaseDataUtils.getHearingWindow(caseData))
```

Comment:
Use:
https://github.com/hmcts/sscs-hearings-api/blob/master/src/main/java/uk/gov/hmcts/reform/sscs/helper/mapping/HearingsCaseMapping.java#L93-L104

Reply:
Think you meant HearingDetailsMapping.buildHearingWindow?

Reply:
Refactored